.storybook/main.js

```diff
@@ -3,22 +3,19 @@ const config = {
   stories: [
     "../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)",
   ],
-  addons: [
-    "@storybook/addon-a11y",
-    {
-      name: "@storybook/addon-essentials",
-      options: {
-        measure: false, // Note: addon-measure is forced to false in order to avoid it enabled in ZeroHeight
-      }
-    },
-    "@storybook/addon-themes",
-  ],
+  addons: ["@storybook/addon-a11y", {
+    name: "@storybook/addon-essentials",
+    options: {
+      measure: false, // Note: addon-measure is forced to false in order to avoid it enabled in ZeroHeight
+    }
+  }, "@storybook/addon-themes",],
   framework: {
     name: "@storybook/html-vite",
     options: {},
   },
   docs: {
-    defaultName: 'Docs'
+    defaultName: 'Docs',
+    autodocs: true
```

Comment:
I just checked a `npx storybook@latest upgrade` on the main branch and didn't get this `autodocs: true`. When I ran `npx storybook automigrate` it added this autodocs, but removed it 2 steps later to add it as a tag in preview.

Since the `autodocs` tag is already set in preview.js, and [this config is deprecated](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mainjs-docsautodocs-is-deprecated), I would suggest to remove this old autodocs configuration here.

Reply:
You're right, I think I said yes to every question, but I shouldn't, I pushed something new. I won't do it anymore, thanks for review ❤️